src/playlist-loader.js

```diff
@@ -227,6 +228,44 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
     }
   };
 
+  /**
+   * Returns the number of enabled playlists on the master playlist object
+   *
+   * @return {Number} number of eneabled playlists
+   */
+  loader.enabledPlaylists = function() {
+    return loader.master.playlists.filter((element, index, array) => {
+      return element.excludeUntil <= Date.now() || !element.excludeUntil ? true : false;
```

Comment:
You can rely on javascript's implicit boolean conversions to avoid the ternary:
```js
element.excludeUntil <= Date.now() || !element.excludeUntil
```
Since your second expression is a pre-requisite of the first, I'd swap the order:
```js
!element.excludeUntil || element.excludeUntil <= Date.now()
```
The `||` operator will short-circuit and skip the date comparison which is nice (but not important), too.